index.php

```diff
@@ -140,6 +149,16 @@
 		}
 	}
 
+	if($mybb->settings['wolorder'] == 'activity')
+	{
+		// activity ordering is DESC, username is ASC
+		krsort($onlinebots);
+	}
+	else
+	{
+		ksort($onlinebots);
+	}
+
```

Comment:
As stated earlier - sorting is already there in query results. So why sorting by key is again required?